html/register-user/available-user.php

```diff
@@ -0,0 +1,17 @@
+<?php
+require_once('../db/connection.php');
+$uId = $_POST['user_id'];
+$res = $conn->query('select * from user');
+$result = "";
+if ($res->num_rows > 0) {
+    while ($row = $res->fetch_assoc()) {
+        if ($uId == $row["user_id"]) {
+            $result = "user_id already exists.";
+            ?>
+            <span class="error">Username not available</span>
+            <?php
+            break;
+        }
+    }
+}
+?>
```

Comment:
This is look like closing tag of 1st line which is not looking good that's why I comment above.